src/System.Diagnostics.Tracing/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingTypeInfo.cs

```diff
@@ -61,7 +61,7 @@ internal TraceLoggingTypeInfo(Type dataType)
 
             if (name == null)
             {
-                throw new ArgumentNullException("eventName");
+                throw new ArgumentNullException(nameof(name));
```

Comment:
The only entry point I found is the `InvokeTypeInfo`'s constructor:
```
public InvokeTypeInfo(
            Type type,
            TypeAnalysis typeAnalysis)
            : base(
                type,
                typeAnalysis.name,
                typeAnalysis.level,
                typeAnalysis.opcode,
                typeAnalysis.keywords,
                typeAnalysis.tags)
```
Anyway, I will revert this in next commit. Let me know, if you want me to make it use `nameof`.